examples/wind_example.ipynb

```diff
@@ -0,0 +1,925 @@
+       "      <td>191</td>\n",
+       "      <td>9.2</td>\n",
+       "      <td>999</td>\n",
+       "      <td>99.0</td>\n",
+       "      <td>9999</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2018-01-11 23:40:00</th>\n",
+       "      <td>196</td>\n",
+       "      <td>7.5</td>\n",
+       "      <td>999</td>\n",
+       "      <td>99.0</td>\n",
+       "      <td>9999</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2018-01-11 23:50:00</th>\n",
+       "      <td>185</td>\n",
+       "      <td>8.7</td>\n",
+       "      <td>174</td>\n",
+       "      <td>12.7</td>\n",
+       "      <td>2253</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "<p>144 rows × 5 columns</p>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                     WDIR  WSPD  GDR   GST  GTIME\n",
+       "date                                             \n",
+       "2018-01-11 00:00:00   199   6.6  999  99.0   9999\n",
+       "2018-01-11 00:10:00   197   6.6  999  99.0   9999\n",
+       "2018-01-11 00:20:00   196   6.1  999  99.0   9999\n",
+       "2018-01-11 00:30:00   194   6.1  999  99.0   9999\n",
+       "2018-01-11 00:40:00   186   6.3  999  99.0   9999\n",
+       "...                   ...   ...  ...   ...    ...\n",
+       "2018-01-11 23:10:00   186  10.2  999  99.0   9999\n",
+       "2018-01-11 23:20:00   192   9.4  999  99.0   9999\n",
+       "2018-01-11 23:30:00   191   9.2  999  99.0   9999\n",
+       "2018-01-11 23:40:00   196   7.5  999  99.0   9999\n",
+       "2018-01-11 23:50:00   185   8.7  174  12.7   2253\n",
+       "\n",
+       "[144 rows x 5 columns]"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Display DataFrame of 46022 data from January 11, 2018\n",
+    "daily_data = ndbc_data['2018']['2018-01-11']\n",
+    "daily_data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The data for January 11, 2018 is given in 10 minute intervals. We can consider the average hourly wind speed to cover the 30 minute periods before and after each hour. For example, the mean speed at 2:00 averages all speeds between 1:30-2:30."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "# Get average hourly wind speeds\n",
+    "hours = np.arange(1,24,1)\n",
+    "hourly_mean_speeds = []\n",
+    "for hour in hours:\n",
+    "    # Create a boolean mask using the start and end times\n",
+    "    start_time = '2018-01-11 ' + str(hour-1) + ':30:00'\n",
+    "    end_time = '2018-01-11 ' + str(hour) + ':30:00'\n",
+    "    mask = (daily_data.index >= start_time) * (daily_data.index < end_time)\n",
+    "    hourly_data = daily_data[mask]\n",
+    "    hourly_mean_speeds.append(np.mean(hourly_data.WSPD))"
```

Comment:
I believe this whole code block is simply :
`daily_data.resample('h').mean()`

Reply:
Thanks! I wasn't aware of this dataFrame method yet. Updated this to use ``resample()``